api/apps/api/src/modules/scenarios/marxan-run/cancel.handler.ts

```diff
@@ -34,12 +35,7 @@ export class CancelHandler {
       return right(void 0);
     }
 
-    if (await scenarioJob.isActive())
-      await scenarioJob.updateProgress({
-        canceled: true,
-        scenarioId,
-      });
-    else if (await scenarioJob.isWaiting()) await scenarioJob.remove();
```

Comment:
the implementation in the scenario-job.service does not account for this "else". Can you double-check that this has no undesirable side effects please?

Reply:
You are right, I completely forgot the return statement in the first if block, thank you 👍